hardware_enablement/psap-special-resource-operator.adoc

```diff
@@ -0,0 +1,24 @@
+[id="special-resource-operator"]
+= The Special Resource Operator
+include::modules/common-attributes.adoc[]
+:context: special-resource-operator
+
+toc::[]
+
+Learn about the Special Resource Operator (SRO) and how you can use it to build and manage driver containers for loading kernel modules and device drivers on nodes in an {product-title} cluster.
+
+
+:FeatureName: The Special Resource Operator
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+include::modules/psap-special-resource-operator.adoc[leveloffset=+1]
+
+include::modules/psap-special-resource-operator-installing.adoc[leveloffset=+1]
+
+include::modules/psap-special-resource-operator-using.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Maybe mention something about SNO not having the image registry, and requiring enable or an external one?

Reply:
I added a pointer to the additional resources in the prerequisites modules/special-resource-operator-using.adoc to make this more clear, but didn't add a specific mention of SNO because it's not specific to SNO. It also applies to baremetal installs, etc. Let me know if you think more clarification is needed